react/props_and_component_structure/README.md

```diff
@@ -19,31 +18,29 @@
 
 Another example of this is Amazon's product search. Take a look at, say, a search for t-shirts:
 
 ![amazon](./assets/amazon_screen.png)
 
-Each of these t-shirts represents a separate instance of the **exact same component**. This component is reused in many different places in the application.
+Each of these t-shirt listings represents a separate instance of the **exact same component**. This component is reused in many different places in the application.
 
 In order to have the same component display different information, we can use **props**.
 
 ## Props
 
-Just like React Components have a `state` property, they also have a `props` property.  Props allow components to render other components and to **pass information to those components**.  Components can use props to accurately display information as the state of its parent changes.
+Props allow components to **pass information** to child components that they're rendering. Components can use props to consume data from their parents and render it to the user. This permits us to reuse the same component with relative ease. For example,we don't need distinct `UserProfile` components if each user's profile consumes the same shape of information and renders that information in the same way. We can use the same component and pass down unique user information as props.
 
-Props also allow us to intelligently divide responsibilities between components. We can create components whose only job is to make AJAX requests and store state, and we can have components whose only job is to receive props and render information.
+Props also allow us to intelligently divide responsibilities between components. We can create components whose only job is to fetch, store, and update information, and we can have components whose only job is to receive props and render information. More on that separation of concerns in a future lesson.
 
-Returning to our Amazon search page example, let's say our components (`ProductItem`) are the *child components* of a component for the entire page (`ProductPage`). `ProductPage` makes one large AJAX request on the first render of the page, getting an array with fifty products. It then sets a part of state (`this.state.products`) with this array. We map through the array and send each individual product to a `ProductItem` component by way of props, which processes the product's information into something user-facing.
+Returning to our Amazon search page example, let's say our components (`ProductItem`) are the *child components* of a component for the entire page (`ProductPage`). `ProductPage` stores the result of a large AJAX request on the first render of the page, getting an array with fifty products. It then sets a part of state (`this.state.products`) with this array. We map through the array and send each individual product to a `ProductItem` component by way of props, which processes the product's information into something user-facing.
```

Comment:
Need to remove this reference to state here. I think in general we could make this explanation a lot simpler. AJAX calls, states, etc. are not directly relevant to the idea of props and one-way data flow from parent to child.

Reply:
I also think a visual here could probably help make the text more digestible.